libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -77,9 +77,12 @@ Aircraft::Aircraft(const char *frame_str) :
 
     // ahrs_orientation->get() returns ROTATION_NONE here, regardless of the actual value
     enum Rotation imu_rotation = ahrs_orientation?(enum Rotation)ahrs_orientation->get():ROTATION_NONE;
-    sitl->ahrs_rotation.from_rotation(imu_rotation);
-    sitl->ahrs_rotation_inv = sitl->ahrs_rotation.transposed();
     last_imu_rotation = imu_rotation;
```

Comment:
Presumbly there is no side-effect on imu_rotation from from_rotation()?

Reply:
the argument of from_rotation isn't declared const, but it's not altered by Vector3f::rotate or Matrix3f::from_rotation